### PR TITLE
Update the docs home page (part 1)

### PR DESCRIPTION
**Pull request by francip**

### CODE REVIEW

**Comment by francip** [comment]
Screenshot 2024-06-08 225109

File: docs/index.md [hunk 1]
```diff
@@ -4,54 +4,36 @@ title: Introduction
...
-linkStyle 1 stroke:#1ABC9C, stroke-width:2px
-linkStyle 2 stroke:#FFCC00, stroke-width:2px
-linkStyle 3 stroke:#2ECC40, stroke-width:2px
-```
+- Open-Source: Fully open-source, empowering you to customize and enhance your experience.
+- Long Battery Life: 24+ hour battery ensures you stay connected all day.
+- Live Transcription: Capture live voice and audio with human-level accuracy using OpenAI Whisper and Deepgram.
+- Conversation Memory: Remembers conversations, providing you with reliable records.
+- Efficient Summarization: Get instant insights and summaries powered by ChatGPT in just 5 seconds.
+- Connectivity: Bluetooth enabled with WiFi, LED, accelerometer, and USB-C charging.
```
Comment (akshaynarisetti) [inline docs/index.md]: We don’t have Wifi, rest features are correct.

File: docs/index.md [hunk 2]
```diff
@@ -4,54 +4,36 @@ title: Introduction
...
-class A,B,C,D,E lightMode
-class A,B,C,D,E darkMode
-
-linkStyle 0 stroke:#FF4136, stroke-width:2px
-linkStyle 1 stroke:#1ABC9C, stroke-width:2px
-linkStyle 2 stroke:#FFCC00, stroke-width:2px
-linkStyle 3 stroke:#2ECC40, stroke-width:2px
-```
+- Open-Source: Fully open-source, empowering you to customize and enhance your experience.
+- Long Battery Life: 24+ hour battery ensures you stay connected all day.
```
Comment (akshaynarisetti) [inline docs/index.md]: The battery lasts 6+ days.

**Review by akshaynarisetti — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (francip) [reply docs/index.md]: Lol, I literally copied this from the description of our iOS app in App Store. :-)

**Review by akshaynarisetti — APPROVED** [review]

**Comment by adamcohenhillel** [comment]
Nice!

**Pull request merged**